packages/camera/camera_windows/windows/task_runner.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef PACKAGES_CAMERA_CAMERA_WINDOWS_WINDOWS_TASK_RUNNER_H_
+#define PACKAGES_CAMERA_CAMERA_WINDOWS_WINDOWS_TASK_RUNNER_H_
+
+#include <functional>
+
+namespace camera_windows {
+
+/// A closure that can be run as a task.
+using TaskClosure = std::function<void()>;
+
+/// A task queue for scheduling functions to be executed in a different thread.
```

Comment:
This needs more discussion. What thread does it run things on? Is it threadsafe? These things seem obvious when writing a class like this, but aren't necessarily obvious to someone encountering this code for the first time.

Reply:
The `TaskRunner` is just an interface here, so from that perspective probably the best we could say here is that the thread is probably different from the calling thread. However, maybe this interface class can be dropped and I’ll use `TaskRunnerWindow` directly. What do you think?